Framework/src/SliceTrendingTask.cxx

```diff
@@ -187,7 +187,7 @@ void SliceTrendingTask::generatePlots()
           beautifyGraph(multigraph, plot, c);
           multigraph->Draw("A pmc plc");
           c->cd(2);
```

Comment:
I think we can remove this. We don't need to switch to the second pad anymore as the legend was already drawn.